ffv1.md

```diff
@@ -26,6 +26,12 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 ## Definitions
 
 -------- --------------------------------------------------------------
+`Container`: Format whose specification describes how packets and metadata coexist in a file.
+
+`Configuration Record`: Encoder-created bitstream encapsulated in a container, representing a meaningful non visual element used for setting up the decoder.
```

Comment:
could consider `initialising the decoder` rather than setting up

Reply:
non-visual?

Reply:
Meaning this is not a frame, no visual info encoded.
Better wording to suggest?

Reply:
Oops, sorry: I have the *feeling* that a `-` is necessary, but I may be wrong.

Reply:
Ha. Ok I'll add the dash

Reply:
added locally, will push to this branch soon